packages/create-sitecore-jss/src/templates/nextjs-personalize/src/components/CdpIntegrationScript.tsx

```diff
@@ -8,7 +8,7 @@ declare const Boxever: any;
 
 function createPageView(locale: string | undefined, routeName: string) {
   // POS must be valid in order to save events (domain name might be taken but it must be defined in CDP settings)
-  const pos = 'spintel.com';
+  const pos = process.env.CDP_POS || window.location.host.replace(/^www\./,'');
 
   _boxeverq.push(function () {
     const pageViewEvent = {
```

Comment:
I got a link with the `pageViewEvent` type, can you define this type?
https://doc.sitecore.com/cdp/en/developers/sitecore-customer-data-platform--data-model-2-1/send-a-view-event-to-the-sitecore-cdp.html